content/en/docs/instrumentation/go/sampling.md

```diff
@@ -1,36 +1,43 @@
 ---
 title: Sampling
 weight: 80
-cSpell:ignore: sdktrace
 ---
 
 Sampling is a process that restricts the amount of traces that are generated by
 a system. The exact sampler you should use depends on your specific needs, but
 in general you should make a decision at the start of a trace, and allow the
 sampling decision to propagate to other services.
 
-A sampler needs to be set on the tracer provider when its configured, as
-follows:
+A [`Sampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#Sampler) can
+to be set on the tracer provider when register it using the
+[`WithSampler`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#WithSampler)
+option, as follows:
 
 ```go
-provider := sdktrace.NewTracerProvider(
-    sdktrace.WithSampler(sdktrace.AlwaysSample()),
+provider := trace.NewTracerProvider(
+    trace.WithSampler(trace.AlwaysSample()),
 )
 ```
 
-`AlwaysSample` and `NeverSample` are fairly self-explanatory. Always means that
-every trace will be sampled, the converse holds as true for Never. When you're
-getting started, or in a development environment, you'll almost always want to
-use `AlwaysSample`.
+[`AlwaysSample`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#AlwaysSample)
+and
+[`NeverSample`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/trace#NeverSample)
+are fairly self-explanatory. Always means that every trace will be sampled, the
+converse holds as true for Never. When you're getting started, or in a
+development environment, you'll almost always want to use `AlwaysSample`.
```

Comment:
```suggestion
are self-explanatory values. `AlwaysSample` means that every trace is
sampled, while `NeverSample` means that no trace is sampled. When
you're getting started, or in a development environment, use
`AlwaysSample`.
```

Reply:
Are there production recommended values? When one would use `NeverSample`?

Reply:
E.g. one someone would like to have it configurable.

Reply:
Could you add a sentence on when would `NeverSample` be useful?